bin/process-and-import-ccd-definition.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eu
+
+scriptPath=$(dirname $(realpath $0))
+
+root_dir=$(realpath $(dirname ${0})/..)
+build_dir=${root_dir}/build/ccd-config
+
+ ${scriptPath}/ccd-build-definition.sh
```

Comment:
Is it pulling the st_cic ccd definition?

Reply:
or is it using another definition?

Reply:
It looks like it's trying to get the definition files from build/definitions like it does in case api but there are not any there. Think I need to look into how our definition files are loaded in case api.

Reply:
Have had a look into this. Looks like there isn't a way to import the ST ccd definitions unless we pull them from case api. Looking into the build further, it looks like it doesn't need the ccd definitions to upload the dmns to camunda so will be removing